add convet function to vultr/v1_reservedip

nickruhl opened the pull request.

dopry commented:
Please add test coverage under the authenticated test suite.

dopry commented:
@nickruhl I rebased you PR on master after I fixed the travis build. You probably want to fetch it and reset your local master to the new remote master.

nickruhl commented:
Thank you. Will look into this today.

On Tue, Feb 4, 2020 at 9:58 PM dopry <notifications@github.com> wrote:

> @nickruhl <https://github.com/nickruhl> I rebased you PR on master after
> I fixed the travis build. You probably want to fetch it and reset your
> local master to the new remote master.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/spry-group/python-vultr/pull/51?email_source=notifications&email_token=AJ333CSNCPZDO5SVN4S33Y3RBJBQZA5CNFSM4KQELLZKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEK2ETTY#issuecomment-582240719>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ333CWEAFLDFUKQNGULOTTRBJBQZANCNFSM4KQELLZA>
> .
>

dopry commented:
@nickruhl have you had a chance to update the tests?

nickruhl commented:
Hi. No I dropped this as I am no longer hosting in Vultr.

On Mon, Aug 9, 2021 at 11:03 AM dopry ***@***.***> wrote:

> @nickruhl <https://github.com/nickruhl> have you had a chance to update
> the tests?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/spry-group/python-vultr/pull/51#issuecomment-895387418>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ333CS5GD3YHVXPDHYTZF3T4AC6VANCNFSM4KQELLZA>
> .
>
